static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -27,7 +28,8 @@ define([
     Giraffe,
```

Comment:
is `Giraffe` and `giraffeHeader` the clearest names for this? I know that your team name is that, but your team name may change and this code may still be here